src/test/OpenEXRTest/testSampleImages.cpp

```diff
@@ -63,7 +63,7 @@ bool
 approximatelyEqual (float x, float y)
 {
     float z = (x + 0.01f) / (y + 0.01f);
-    return z >= 0.99f && z <= 1.01f;
+    return z >= 0.9f && z <= 1.1f;
```

Comment:
haha why did tolerances go from "suck" to "super suck"?

Reply:
because we weren't actually comparing the files previously - it was just comparing image a to image a, not image a to image b, and ratios is not a good mechanism to compare image diffs :D